ftdetect/minisnip.vim

```diff
@@ -0,0 +1,13 @@
+" Vim ftdetect file
+" Language:      minisnip
+" Created by:    Robert Hicks <gowther@zoho.com>
+" Last Change:   2017-08-20
+"
+" This simply sets all files that are in the defined
+" directory as 'minisnip'.
+"
+
+" get the snippets directory
+let snippet_dir = expand(g:minisnip_dir) . "*"
+
+execute 'autocmd BufRead,BufNewFile ' . snippet_dir . ' set filetype=minisnip'
```

Comment:
On Windows at work, I have to explicitly set g:minisnip_home because my $HOME is a network drive but Vim is installed under C:\. Not sure how to get around that. I was hoping just put the package in place and everything works.

Try:

" get the snippets directory
"let snippet_dir = expand(g:minisnip_dir) . "*"

"execute 'autocmd BufRead,BufNewFile ' . snippet_dir . ' set filetype=minisnip'
execute "autocmd BufRead,BufNewFile /" g:minisnip_dir . '*' "/ set filetype=minisnip"

Still looking but maybe just put the syntax one in place with an addition to the docs to say if you want syntax highlighting while editing your snippets do:

autocmd BufRead,BufNewFile /path/to/snippets set filetype=minisnip